scripts/install_from_release.sh

```diff
@@ -12,13 +12,42 @@ fi
 
 # Map the language parameter to the corresponding scenarios file
 # If no parameter, download the scenarios from IE
-if [ "$LANG" = "" ]; then
+MAIN_LANG_PREFIX="$(echo "$LANG" | head -c2 | tr '[:upper:]' '[:lower:]')"
+LANG_ARRAY_1=("de" "es" "fr" "it" "nl" "pt")
+LANG_ARRAY_2=("cs" "hu" "id" "ja" "ko" "pl" "pt" "ru" "sv" "tr")
+
+if [ "$MAIN_LANG_PREFIX" = "" ] || [ "$MAIN_LANG_PREFIX" = "en" ]; then
   SCENARIOS="https://github.com/Azure/InnovationEngine/releases/download/$RELEASE/scenarios.zip"
-# Otherwise, download the scenarios from Microsoft Docs in the appropriate langauge
-elif [ "$LANG" = "en-us" ]; then
-  SCENARIOS='https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/scenarios.zip'
+elif [ "$MAIN_LANG_PREFIX" = "pt" ]; then
+  if [ "$LANG" = 'pt-pt' ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/pt-pt-scenarios.zip"
+  elif [ "$LANG" = 'pt-br' ]; then
+    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/pt-br-scenarios.zip"
+  fi
```

Comment:
We need an else case here to download pt-pt incase the code is something else 'pt-xx'
```suggestion
  else
    SCENARIOS="https://github.com/MicrosoftDocs/executable-docs/releases/download/v1.0.1/pt-pt-scenarios.zip"
  fi
```